src/cpr_data_access/parser_models.py

```diff
@@ -17,6 +17,11 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+PARSER_METADATA_KEY = "parser_metadata"
+AZURE_API_VERSION_KEY = "azure_api_version"
+AZURE_MODEL_ID_KEY = "azure_model_id"
+PARSING_DATE_KEY = "parsing_date"
```

Comment:
why are these added? it doesn't look like there are any changes in the code to use them anywhere

Reply:
Good question! 

These keys are used in the parser as well as the identify updates stage so I wanted to make sure we were definitely using the right keys.

I.e. in the identify updates we are checking that those keys exist in the `pipeline_metadata` field, thus adding them so we can check my_dict[PARSER_METADATA_KEY] rather than my_dict["parser_metadata"]. 

Let me know if you think there's a better solution using Enum or somthing.

Reply:
this makes sense to me !